app/validators/form_password_validator.rb

```diff
@@ -7,7 +7,7 @@ module FormPasswordValidator
 
     validates :password,
               presence: true,
-              length: Devise.password_length
+              length: { in: Devise.password_length }
```

Comment:
What exactly is the purpose of this change? It does not look like this changes anything?

Reply:
If I'm understanding correctly -- and I may not be -- this will make it confirm that it's _within_ the range, so the 8/9 character mismatch will sort out. But I could be looking at it wrong.

Reply:
Yeah, I think what we have now is a shorthand for that. I don't think the problem is with the Devise validation, but with the zxcvbn password score on the frontend.

So the issue is not length, but rather lack of complexity.